lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>Je obvious, že se to propíše do všech řádků? Asi jo... jen já to teda chápal hlavně díky tomu, že to vim, tak přemýšlím, jestli to tam napsat explicitně.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<blockquote>Explicit is better than implicit.</blockquote><p> </p><p>Dávám to tam.</p>

Reply:
👍